### ISSUE TITLE
Make the File Tree functionality easier to discover and use

### ISSUE THREAD

**Issue by jankrassnigg** [ForkIssues/TrackerWin#919]
The file tree feature is quite useful, especially now that you can look at the history of entire folders. However, it is hard to find, and you need to go through a commit to access it. Here is an idea how to make it more accessible:

![image](https://user-images.githubusercontent.com/6001174/90955184-b232b780-e47b-11ea-8250-15ff50935620.png)

Add a file tree icon next to the branch and search button, to swap out the left hand pane. Show a file tree pane on the left in this mode. Ie. the yellow box. Also replace the right hand pane (the turquoise box) and show the file content instead (the blue box).

That would make the file tree easily accessible. Maybe also add a file filter search box to the file tree, as you already have in the "Changes" file view.

**Comment by DanPristupov** [comment]
The file tree show the repository content **at** the selected commit and logically belongs to other commit details (message and changes). Showing commit details in two places (at the bottom and on the sidebar) will definitely confuse people.

**Comment by JeReT** [comment]
I agree with @jankrassnigg.
The file tree is super useful. I actually was not aware that it existed until now because I thought the stuff below the commit history always just shows changes of the commit.
Until now I always went the way over Ctrl+P if I wanted to check information about a certain file. So, actually the current design confused me.

I would find it way more intuitive having the file tree as a static UI as proposed on the left showing the currently checked out state by default. You may consider to add an option to show the selected branch`s file tree instead (since you already have this functionality) But in most of the cases you just want to go to a file and open the file history or blame - independent from commits.

**Comment by TomGit27** [comment]
I agree with Dan, as the content of the file tree relates to the content at the selected commit. However, there is another way to do this. As Jan suggested having a file tree aside of the branch view is a nice idea. Imo, the solution is to view the current workspace. That is the current checkout. Thus the title could be "Workspace".

**Comment by jankrassnigg** [comment]
What @TomGit27 proposes is what I meant in the first place, in case that wasn't clear. With the current design I pretty much never use the file tree, because it's too inconvenient to find and easily forgotten that the functionality exists at all.

**Comment by doneykoo** [comment]
Vote +1!
Very agreed.
* A dedicated `file tree` panel, which bind to the current checked out working set.

Additionally, my suggestion is:
* And/Or, if user can add new `file tree` panels by our own (e.g. like adding more tabs beside the current), and able to ***lock*** it (locking it to the dedicated selected commit id, or, branch)

, would be much better helping.

Reason: 
* My focusing & working progress to the current file tree pane could very easily be lost, when you click around revealing some history commits of some files or folders that you are looking into.
* The problem in it is that we only have the current files tree panel which is binding to the actively selected single commit log, the file tree files immediately changes to new states whenever I select another commit id, no longer the one I was looking into.
